dockers/docker-dhcp-relay/Dockerfile.j2

```diff
@@ -1,26 +1,11 @@
-FROM docker-base
+FROM docker-config-engine
 
 ## Make apt-get non-interactive
 ENV DEBIAN_FRONTEND=noninteractive
 
 ## Update APT package lists
 RUN apt-get update
 
-## Dependencies for sonic-cfggen
```

Comment:
I'll vote for keep those lines as removing them will cause confusion when later developers want to add dependency for this docker.
If we really want to remove those lines, maybe revert this docker from DOCKER_IMAGES back to SIMPLE_DOCKER_IMAGES in rule.mk?